contracts/optimistic-ethereum/mockOVM/bridge/mockOVM_CrossDomainMessenger.sol

```diff
@@ -32,6 +32,8 @@ contract mockOVM_CrossDomainMessenger is OVM_BaseCrossDomainMessenger {
     address internal targetMessengerAddress;
     uint256 internal lastRelayedMessage;
     uint256 internal delay;
+    uint256 public messageNonce;
+    address override public xDomainMessageSender;
```

Comment:
What is this override for? Is this to make xDomainMessageSender public?

Reply:
because it's defined in the interface!